### DISCUSSION TITLE
Picture and item frames are not protected

### DISCUSSION THREAD

**Discussion by Pencil1098** [GriefPrevention/GriefPrevention#2154] [Q&A]
Pictures and item frames are not protected.
[config - 복사본.zip](https://github.com/TechFortress/GriefPrevention/files/12584126/config.-.zip)

**Comment by QarthO** [comment]
Please follow the template when creating an issue. No one will download your zip file.

Picture and item frames are protected in pretty much all of the recent versions. Tested and confirmed. So there is a good chance the issue has something to do with your setup, or you have a conflicting plugin.
